C/tests/c4DatabaseTest.cc

```diff
@@ -526,6 +535,92 @@ class C4DatabaseTest : public C4Test {
         AssertEqual(seq, (C4SequenceNumber)100);
     }
 
+    void testExpired() {
+        C4Slice docID = C4STR("expire_me");
+        createRev(docID, kRevID, kBody);
+        time_t expire = time(NULL) + 1;
+        C4Error err;
+        Assert(c4doc_setExpiration(db, docID, expire, &err));
+        
+        expire = time(NULL) + 2;
+        // Make sure setting it to the same is also true
+        Assert(c4doc_setExpiration(db, docID, expire, &err));
+        Assert(c4doc_setExpiration(db, docID, expire, &err));
+        
+        C4Slice docID2 = C4STR("expire_me_too");
+        createRev(docID2, kRevID, kBody);
+        Assert(c4doc_setExpiration(db, docID2, expire, &err));
+    
+        C4Slice docID3 = C4STR("dont_expire_me");
+        createRev(docID3, kRevID, kBody);
+        sleep(2u);
+        
+        auto e = c4db_enumerateExpired(db, &err);
+        Assert(e != NULL);
+        
+        int expiredCount = 0;
+        C4DocumentInfo info;
+        while(c4exp_next(e, NULL)) {
+            c4exp_getInfo(e, &info);
```

Comment:
You removed `c4exp_getInfo` in the latest commit, so you'll need to update this test too. Right now it breaks the build.